packages/oas/src/converter/parts/headers/HeadersConverter.ts

```diff
@@ -35,31 +45,53 @@ export class HeadersConverter implements SubConverter<Header[]> {
     return headers;
   }
 
-  private createContentTypeHeaders(pathObj: OperationObject): Header[] {
-    const values = [];
-
-    if ('consumes' in pathObj && Array.isArray(pathObj.consumes)) {
-      values.push(...pathObj.consumes);
-    } else if (
-      'requestBody' in pathObj &&
-      'content' in pathObj.requestBody &&
-      pathObj.requestBody?.content
-    ) {
-      values.push(...Object.keys(pathObj.requestBody.content));
+  protected parseApiKeyScheme(
+    securityScheme: SecuritySchemeObject
+  ): Header | undefined {
+    if ('in' in securityScheme && securityScheme.in === 'header') {
+      return this.createApiKeyHeader(securityScheme);
     }
+  }
+
+  protected createBearerAuthHeader(): Header {
+    return this.createHeader('authorization', 'Bearer REPLACE_BEARER_TOKEN');
```

Comment:
Why don't use the sampler here?

```suggestion
    const token = this.sampler.sample({ type: 'string', maxLength: 10, minLength: 5 });
    
    return this.createHeader('authorization', `Bearer ${token}`);
```